packages/neo-one-website/src/components/docs/Subsection.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { Link as RouterLink } from 'react-static';
+import { styled } from 'reakit';
+import { prop } from 'styled-tools';
+
+export interface Props {
+  readonly slug: string;
+  readonly title: string;
+  readonly onClick?: () => void;
+}
+
+const NavigationLink = styled(RouterLink)`
+  display: flex;
+  align-items: center;
+  ${prop('theme.fonts.axiformaRegular')};
+  font-size: 16px;
+  height: 100%;
+  width: 100%;
+  padding-top: 8px;
+  color: ${prop('theme.black')};
+  text-decoration: none;
+
+  &:hover {
+    color: ${prop('theme.accent')};
+  }
+
+  &:focus {
+    ${prop('theme.fonts.axiformaBold')};
+  }
+`;
+
+export const Subsection = ({ slug, title, onClick }: Props) => {
```

Comment:
All of your components should follow this pattern:

```
export const Component = ({ prop0, prop1, ...props }: Props) => (
  <Wrapper {...props}>
    ...
  </Wrapper>
);
```

So that they can be easily styled wherever they're used. (Note `Wrapper` isn't necessarily what it needs to be called, e.g. in this component using `li` is correct)